decidim-core/config/locales/en.yml

```diff
@@ -402,13 +402,13 @@ en:
           affected_user:
             email_intro: 'A new amendment has been created for %{amendable_title}. You can see it from this page:'
             email_outro: You have received this notification because you are an author of %{amendable_title}.
-            email_subject: New amendment for %{amendable_title} from %{emendation_author_nickname}
-            notification_title: A <a href="%{emendation_path}">new amendment</a> has been created by <a href="%{emendation_author_path}">%{emendation_author_nickname}</a> for <a href="%{amendable_path}">%{amendable_title}</a> %{amendable_type}.
+            email_subject: New amendment for %{amendable_title}
+            notification_title: A new amendment has been created for the %{amendable_type} <a href="%{amendable_path}">%{amendable_title}</a>.
```

Comment:
Adding the amendable type here will make it hard to translate in gendered languages like Spanish/Catalan, is it really needed? Alternatively, maybe changing the the sentence to something like this is clearer?

```
A new amendment has been created for <a href="%{amendable_path}">%{amendable_title}</a> (%{amendable_type})
```

Same for the other lines.

Reply:
Thanks

Reply:
@mrcasals I opted for only using `amendable_type` in `promote` notifications, where it really makes sense:
>A rejected amendment for #{amendable_title} has been published as a new #{amendable_type}.

Reply:
Is it okay?

Reply:
@mrcasals is this okay?

Reply:
Whoops, missed this notification, sorry!

Yeah, I guess we can't solve the issue easily, so 👍